src/behaviors/focusTrap.ts

```diff
@@ -64,55 +64,51 @@ export function focusTrap(
 
   container.setAttribute('data-focus-trap', 'active')
   let lastFocusedChild: HTMLElement | undefined = undefined
-  const firstFocusableChild = getFocusableChild(container)
-  const lastFocusableChild = getFocusableChild(container, true)
 
   // Ensure focus remains in the trap zone by checking that a given recently-focused
   // element is inside the trap zone. If it isn't, redirect focus to a suitable
   // element within the trap zone. If need to redirect focus and a suitable element
   // is not found, return false.
```

Comment:
this comment needs to be updated (no longer returns a boolean)

Reply:
Great catch, updated